src/components/security_manager/src/security_query.cc

```diff
@@ -172,7 +172,10 @@ const SecurityQuery::QueryHeader& SecurityQuery::get_header() const {
 }
 
 const uint8_t* SecurityQuery::get_data() const {
-  return &data_[0];
+  if (!data_.empty()) {
```

Comment:
@AGaliuzov , when we just creates SwqurityQuery object, and trying to query.get_data() we will get out of range.